File: src/Students.java
import java.util.HashMap;

public class Students {

    static HashMap<Integer, Student> students;

    public static void initialize() {
        students = new HashMap<>();
    }

    public static boolean exists(int id) {
        return students.containsKey(id);
    }

    public static Student add(int number, Student student) {
        return students.put(number, student);
    }

    public static Student get(int number) {
        return students.get(number);
    }

}

File: src/Students.kt
package io.github.potatocurry.kys

/** Holds [Student] objects and acts as interface for accessing database. */
object Students {
    private var students = HashMap<Int, Student>()

    /** Clears database. */
    fun clear() {
        students.clear()
    }

    /** Adds a new [student] with [number] to the database. */
    fun add(number: Int, student: Student): Student {
        students[number] = student
        return student
    }

    /** Returns the specified [Student] by [number]. */
    operator fun get(number: Int): Student? {
        return students[number]
    }

    /** Returns a random [Student] number. */
    fun getRandom(): Int {
        return students.keys.random()
    }
}
